resources/assets/css/coreuiv2.css

```
@@ -13,3 +13,14 @@ h1[bp-section=page-heading] {
     font-size: 2rem;
     line-height: 2.1rem;
 }
+
+
+.container-repeatable-elements .controls button {
+    background-color: #e8ebf0 !important;
+    height: 30px !important;
+    width: 30px !important;
```

I think 30px is too big. The rest of the buttons inside the form are smaller, to highlight the fact that they're only used inside the form (visual hierarchy) and they're less important than the buttons at the bottom (save actions).

```suggestion
    height: 24px !important;
    width: 24px !important;
    margin-left: 4px!important;
```